README.md

```diff
@@ -228,13 +240,43 @@ In this section, we'll add the necessary configuration to work with GitHub Actio
     }
     ```
 
-    Add a similar function for each example that you want to run in an integration test.  For examples written in other languages, create similar files for `examples_${LANGUAGE}_test.go`.
-1. Ensure that any required secrets (API keys, etc.) are present in GitHub.
+1. Add a similar function for each example that you want to run in an integration test.  For examples written in other languages, create similar files for `examples_${LANGUAGE}_test.go`.
+
+1. You can run these tests locally via Make:
+
+    ```bash
+    make test
+    ```
+
+    You can also run each test file separately via test tags:
+
+    ```bash
+    cd examples && go test -v -tags=nodejs
+    ```
+
+## Configuring CI with GitHub Actions
+
+In this section, we'll add the necessary configuration to work with GitHub Actions for Pulumi's standard CI/CD workflows for providers.
+
+1. Generate GitHub workflows per [the instructions in the ci-cmgmt repository](https://github.com/pulumi/ci-mgmt/#readme) and copy to `.github/` in this repository.
```

Comment:
`ci-mgmt`.  I'm a little wary of duplicating the text here just because it may get out of sync if we change `ci-mgmt`'s README.  Feel free to ignore.